UI/renderer.js

```diff
@@ -79,19 +79,20 @@ function openSerialPort()
     console.log("Opening serial port: ", e.options[e.selectedIndex].value);
     port = new serialport(e.options[e.selectedIndex].value, { baudRate: 115200 }, function (err) {
         if (err) {
-          return console.log('Error: ', err.message)
+          window.alert(`Error while opening serial port: ${err.message}`);
+          throw err;
```

Comment:
Just to show user potential `Access denied` on linux. Re-throwing will make stack trace available in console